src/Sql/dbo/Stored Procedures/OrganizationUser_ReadByUserIdWithPolicyDetails.sql

```diff
@@ -0,0 +1,40 @@
+﻿CREATE PROCEDURE [dbo].[OrganizationUser_ReadByUserIdWithPolicyDetails]
```

Comment:
I already said this above, but please delete the old sproc if we're no longer using it.

Reply:
Followed instructions on https://contributing.bitwarden.com/contributing/database-migrations/#non-backwards-compatible-migration